core/dbt/contracts/graph/nodes.py

```diff
@@ -532,7 +567,7 @@ def depends_on_nodes(self):
 
     @property
     def depends_on_macros(self):
-        return []
+        return self.depends_on.macros
```

Comment:
Can we add a type annotation for the return? I spend a lot of time digging through code to see what object I'm getting downstream. That would make it easier, and maybe even save me a few click-throughs.

Reply:
Agreed. Adding it everywhere could turn into a full time job. I'm just interested in stopping the bleeding for now.